packages/nextra-theme-docs/src/index.tsx

```diff
@@ -173,28 +173,27 @@ const Content: React.FC<LayoutProps> = ({
 
   const headingArr = headings ?? []
   return (
-    <React.Fragment>
-      <Head title={title} locale={locale} meta={meta} />
       <MenuContext.Provider
         value={{
           menu,
           setMenu,
           defaultMenuCollapsed: !!config.defaultMenuCollapsed
         }}
       >
+        <Head title={title} locale={locale} meta={meta} />
         <div
           className={cn('nextra-container main-container flex flex-col', {
             rtl: isRTL,
             'menu-active': menu
           })}
         >
-          {themeContext.navbar ? (
+          {themeContext.navbar && (
             <Navbar
               isRTL={isRTL}
               flatDirectories={flatDirectories}
               items={topLevelPageItems}
             />
-          ) : null}
+          )}
```

Comment:
I prefer `?` over `&&` because if the condition `themeContext.navbar` is number `0` then React will render "0" here.

Reply:
@shuding fixed for all modified cases, also I fixed types for PageTheme 
<img width="843" alt="image" src="https://user-images.githubusercontent.com/7361780/180612694-78449e2c-aeca-4f76-9b36-020b07a78bd3.png">
since like regression by @promer94 rebase